### PR TITLE
JAMES-3435 Add missing event store module for CassandraMailboxMapperC…

### PR DESCRIPTION
**Pull request by Arsnael**
…oncurrencyTest

Master branch is broken, this is the fix :) Thanks @LanKhuat for the report

### CODE REVIEW

**Review by LanKhuat — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperConcurrencyTest.java [hunk 1]
```diff
@@ -54,7 +55,8 @@
             CassandraBlobModule.MODULE,
             CassandraSchemaVersionModule.MODULE,
             CassandraMailboxModule.MODULE,
-            CassandraAclModule.MODULE));
+            CassandraAclModule.MODULE,
+            CassandraEventStoreModule.MODULE()));
```
Comment (LanKhuat) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperConcurrencyTest.java:59]: Do we need the brackets?

Comment (chibenwa) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperConcurrencyTest.java:59]: Yes (scala)

Comment (Arsnael) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperConcurrencyTest.java:59]: Yes, it's a scala module... Seems we do need those the way it was declared ;)

**Comment by chibenwa** [comment]
test this please

**Comment by LanKhuat** [comment]
test this please

**Pull request merged**